packages/http/src/mocker/negotiator/InternalHelpers.ts

```diff
@@ -19,7 +19,9 @@ export function findExampleByKey(httpContent: IHttpContent, exampleKey: string)
 }
 
 export function hasContents(v: IHttpOperationResponse): v is PickRequired<IHttpOperationResponse, 'contents'> {
-  return !!v.contents;
+  const x = v.contents || [];
+
+  return !!x.filter(y => !y.mediaType.includes('*/*')).length;
```

Comment:
We're going to remove this hack when http-spec will be fixed, right?

Reply:
Yes :)